example/full-config.php

```diff
@@ -58,6 +58,8 @@
                 ],
                 'sql_logger' => null,
                 'middlewares' => [
+                    // List of middlewares doctrine will use to decorate the `Driver` component.
+                    // (see https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/architecture.html#middlewares)
```

Comment:
Couldn't find a static link to the current latest version (only for version 3.2, https://www.doctrine-project.org/projects/doctrine-dbal/en/3.2/reference/architecture.html#middlewares), is this OK?

Reply:
We can link the RST via absolute sha1 reference perhaps? https://github.com/doctrine/dbal/blob/baf6efed70e46f4d979e5dd42f804007e75f12fb/docs/en/reference/architecture.rst#middlewares

Reply:
Perfect, updated the documentation link with the tag of latest release (3.3.2): https://github.com/doctrine/dbal/blob/3.3.2/docs/en/reference/architecture.rst#middlewares